app/jobs/autotitle_conversation_job.rb

```diff
@@ -1,11 +1,16 @@
+class ConverstionNotReady < StandardError; end
+
 class AutotitleConversationJob < ApplicationJob
   queue_as :default
+  retry_on ConverstionNotReady
```

Comment:
Apparently the retry_on defaults to 3.second delay and up to 5 retries. Those felt like reasonable defaults.